### PR TITLE
Kyles additions

### PR DESCRIPTION
**Pull request by kyleatfine**

### CODE REVIEW

**Comment by jamesmk** [comment]
@tshedor I agree. Although maybe the TOC isn't the best place. @kyleatfine on the sections that were were lifted, can you just add `*Taken from [AirBnB's JavaScript Standards](https://github.com/airbnb/javascript)*` at the top?

**Comment by kyleatfine** [comment]
Sure.

**Comment by kyleatfine** [comment]
Updated

File: javascript.md [hunk 1]
```diff
@@ -27,6 +27,490 @@
 
 ## Structure
 
+<<<<<<< HEAD
```
Comment (jamesmk) [inline javascript.md]: Looks like you still have conflict notation in here.

**Pull request merged**